src/layouts/home-assistant.ts

```diff
@@ -93,6 +96,21 @@ export class HomeAssistantAppEl extends HassElement {
     );
     document.addEventListener("freeze", () => this._suspendApp());
     document.addEventListener("resume", () => this._checkVisibility());
+    document.addEventListener("keydown", (e: KeyboardEvent) => {
```

Comment:
Shouldn't this be done by the mixin ? And any reason we wouldn't just open it directly ?

Reply:
Ahh. Yeah I'll try putting it in the mixin. I put it here when I was chasing a red herring where keydown wouldn't register unless I was tabbed into an element.

But IIRC the issue was just that I wasn't attaching it to document.

Reply:
Done

Reply:
Done